src/featureFlagOverrides.service.js

```diff
@@ -45,6 +45,7 @@ angular.module('feature-flags').service('featureFlagOverrides', function($rootEl
       if (localStorageAvailable) {
         return localStorage.getItem(prefixedKeyFor(flagName));
       }
+      return null;
```

Comment:
why does null work but not `return`? or `'return '';`

Reply:
`return` doesn't work with the message `Expected a return value.`

empty string `''` is falsy like you said, but it ends up flowing through this:
```
var value = get(key);
      return typeof value !== 'undefined' && value !== null;
```
which is easy enough to change. I don't know if it's better than `null`?

Reply:
Good to know, happy to leave it as returning null. Looks like the LocalStorage getItem will return null anyway https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem#Return_value